linalg/densemat.cpp

```diff
@@ -401,7 +401,7 @@ void DenseMatrix::InvSymmetricScaling(const Vector & s)
 {
    if (height != width || s.Size() != width)
    {
-      mfem_error("DenseMatrix::SymmetricScaling");
+      mfem_error("DenseMatrix::SymmetricScaling: dimension mismatch");
```

Comment:
Should probably read `DenseMatrix::InvSymmetricScaling: dimension mismatch`

Reply:
Done.